### PR TITLE
feat(exp): Added info about new gear system on experimental

### PR DESCRIPTION
**Pull request by crocket63**
## Summary

Info about new gear feature on experimental

### Location

Discord username (if different from GitHub):

### CODE REVIEW

File: docs/fbw-a32nx/support/exp.md [hunk 1]
```diff
@@ -14,6 +14,20 @@ thread to report any issues - [How to Report Issues](#how-to-report-issues).
 
 ## Implemented Features for Testing
 
+### Hydraulic gear system
+
+- Rigid body physics of doors and gears
+- Dual LGCIU computers implementation with possible failures
+    - Cycling gear handle will switch over to the other LGCIU if available
+- Landging gear lever baulk lock mechanism (both unpowered lgcius will lock lever down)
```
Comment (Valastiri) [inline docs/fbw-a32nx/support/exp.md]:
```suggestion
- Landing gear lever baulk lock mechanism (both unpowered LGCUIs will lock lever down)
```
Also is it baulk or balk?

File: docs/fbw-a32nx/support/exp.md [hunk 2]
```diff
@@ -14,6 +14,20 @@ thread to report any issues - [How to Report Issues](#how-to-report-issues).
 
 ## Implemented Features for Testing
 
+### Hydraulic gear system
```
Comment (Valastiri) [inline docs/fbw-a32nx/support/exp.md]:
```suggestion
### Hydraulic Gear System
```

File: docs/fbw-a32nx/support/exp.md [hunk 3]
```diff
@@ -77,6 +91,14 @@ These features are not yet available but are generally planned and might be impl
 
 ## Known Issues
 
+### Hydraulic gear system
```
Comment (Valastiri) [inline docs/fbw-a32nx/support/exp.md]:
```suggestion
### Hydraulic Gear System Issues
```

**Review by Valastiri — CHANGES_REQUESTED** [review]

File: docs/fbw-a32nx/support/exp.md (hunk 1 above)
Comment (crocket63) [reply docs/fbw-a32nx/support/exp.md]: baulk is good as the term used in all docs, but don't ask me what it exactly means :D

File: docs/fbw-a32nx/support/exp.md [hunk 4]
```diff
@@ -14,6 +14,20 @@ thread to report any issues - [How to Report Issues](#how-to-report-issues).
 
 ## Implemented Features for Testing
 
+### Hydraulic Gear System
+
+- Rigid body physics of doors and gears
+- Dual LGCIU computers implementation with possible failures
+    - Cycling gear handle will switch over to the other LGCIU if available
+- Landing gear lever baulk lock mechanism (both unpowered LGCUIs will lock lever down)
```
Comment (Valastiri) [inline docs/fbw-a32nx/support/exp.md]:
```suggestion
- Landing gear lever baulk lock mechanism (both unpowered LGCIUs will lock lever down)
```

Comment (Valastiri) [reply docs/fbw-a32nx/support/exp.md]: Small typo in my original suggestion will commit this

**Review by Valastiri — APPROVED** [review]

**Comment by Valastiri** [comment]
check build

**Pull request merged**